mappings/net/minecraft/block/PointedDripstoneBlock.mapping

```diff
@@ -8,11 +8,36 @@ CLASS net/minecraft/class_5689 net/minecraft/block/PointedDripstoneBlock
 	FIELD field_28056 FRUSTUM_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28057 MIDDLE_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28058 BASE_SHAPE Lnet/minecraft/class_265;
+	METHOD method_32767 getDripPos (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		ARG 0 world
+		ARG 1 pos
 	METHOD method_32768 createParticle (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)V
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 state
-	METHOD method_32774 isPointedDripstone (Lnet/minecraft/class_2680;Lnet/minecraft/class_2350;)Z
+	METHOD method_32769 getCauldronPos (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_3611;)Lnet/minecraft/class_2338;
+		ARG 0 world
+		ARG 1 pos
+		ARG 2 fluid
+	METHOD method_32770 getThickness (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;Z)Lnet/minecraft/class_5691;
+		ARG 0 world
+		ARG 1 pos
+		ARG 2 direction
+		ARG 3 tryMerge
+	METHOD method_32771 spawnFallingBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)V
+		ARG 0 state
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_32772 dripTick (Lnet/minecraft/class_2680;Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;F)V
+		ARG 0 state
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 dripChance
+	METHOD method_32773 scheduleFall (Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)V
+		ARG 1 state
+		ARG 2 world
+		ARG 3 pos
+	METHOD method_32774 isPointedDripstoneOfDirection (Lnet/minecraft/class_2680;Lnet/minecraft/class_2350;)Z
```

Comment:
```suggestion
	METHOD method_32774 isPointedDripstoneFacingDirection (Lnet/minecraft/class_2680;Lnet/minecraft/class_2350;)Z
```

Reply:
It's probably fine to elide the `PointedDripstone` prefix here altogether since that should be clear from the context